### ISSUE TITLE
DocetLocator > use java.nio.Path instead of java.io.File

### ISSUE THREAD

**Issue by mcRoot** [diennea/docetproject#13]
In general make sure to always use everywhere java.nio library in Docet APIs.

**Comment by mcRoot** [comment]
Using only nio2 Path permit to directly query a zip file without unpacking it on FileSystem. The trick is opening a FileSystem on a zip file:

```
FileSystem fs = FileSystems.newFileSystem(zip, Thread.currentThread().getContextClassLoader())
```
 
See attached test file for examples
[ZipFileSystemAccess.java.txt](https://github.com/diennea/docetproject/files/1087802/ZipFileSystemAccess.java.txt)

**Comment by eolivelli** [comment]
We should add this in 1.8.0 as we are postponing the release

**Comment by diegosalvi** [comment]
Resolved for 1.9.0
